cmd/install.go

```diff
@@ -26,7 +25,11 @@ func execInstallCmd(cmd *cobra.Command, args []string) {
 	useVersionAsDefault, _ := cmd.Flags().GetBool("default")
 	var version string
 	if len(args) == 0 {
-		version = utils.GetLatestVersion()
+		var err error
+		version, err = utils.GetLatestVersion()
+		if err != nil {
+			log.Fatalf(err.Error())
```

Comment:
```suggestion
			log.Fatal(err)
```

log delegates to fmt, and fmt turns errors into strings by calling `.Error()`

```
4. If an operand implements the error interface, the Error method will be invoked to convert the object to a string, which will then be formatted as required by the verb (if any).
```